testing/run_test_suite.sh

```diff
@@ -20,7 +20,7 @@ MYDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source ${MYDIR}/variables.sh
 
 # Coding style check.
-php-cs-fixer fix --diff
+#php-cs-fixer fix --diff
```

Comment:
add this flag to `php_cs.dist`: https://github.com/GoogleCloudPlatform/php-docs-samples/pull/406/files